lib/evmone/eof.hpp

```diff
@@ -43,6 +61,11 @@ struct EOF1Header
     /// The EOF version, 0 means legacy code.
     uint8_t version = 0;
 
+    /// Size of every type section.
+    uint16_t type_section_size = 0;
```

Comment:
We don't need to store the size at all, because header validation guarantees that `type_section_size == code_sizes.siz() * 4`